Lib/test/test_fstring.py

```diff
@@ -70,6 +70,253 @@ def __call__(self):
         # Make sure x was called.
         self.assertTrue(x.called)
 
```

Comment:
How about some test that checks the output of eval()? This code doesn't work as-is, because the NameError doesn't include the line number. But I assume there's a way to capture it and inspect it.

I suggest this because this is the behavior that the end user really cares about.

```
    def test_error_line_numbers(self):
        expr = '''f"""
some text
other text
{x}
"""'''
        with self.assertRaisesRegex(NameError, ', line 5,'):
            eval(expr)
```

Reply:
Wait, never mind. I was conflating this with bpo-31140.